### PR TITLE
Fixed layout issues of profile page

### PR DESCRIPTION
**Pull request by amir-qayyum-khan**
#### What are the relevant tickets?
- fixes https://github.com/mitodl/micromasters/issues/2059 

screen shot 2016-12-26 at 6 22 16 pm

### CODE REVIEW

**Comment by noisecapella** [comment]
Is this ready for review?

**Comment by amir-qayyum-khan** [comment]
yes @noisecapella

File: static/scss/profile-form.scss [hunk 1]
```diff
@@ -23,6 +23,7 @@
   .alert-info {
     background-color: $text-highlight;
     font-style: italic;
+    color: $font-black;
```
Comment (noisecapella) [inline static/scss/profile-form.scss]: What's this change for?

Comment (noisecapella) [inline static/scss/profile-form.scss]: Oh, just read the issue

**Pull request merged**